packages/gatsby-source-medium/src/gatsby-node.js

```diff
@@ -77,10 +76,7 @@ exports.sourceNodes = async (
     resources.map(resource => {
       convertTimestamps(resource)
 
-      const digest = crypto
-        .createHash(`md5`)
-        .update(JSON.stringify(resource))
-        .digest(`hex`)
+      const contentDigest = createContentDigest(JSON.stringify(resource))
```

Comment:
The `createContentDigest` actually uses stringify if it receives a non-string!

So we can tweak this to:

```suggestion
      const contentDigest = createContentDigest(resource)
```